content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,317 @@
+
+### Set clear campaign limits
+
+Create a new feature release timeline with clear parameters and an appropriate ramp down plan:
+
+- **Define why**
+  - System triggers, e.g. a new feature released, a user should not need to dismiss the same annoucement each time they visit a different repository.
+  - User triggers, e.g. a user visits a specific repository for the first time, if recurring, a user should feel added value each time they see the message.
+  - Consider how triggers may be tallied for users who belong to multiple repositories and organizations.
+- **Define when**
+  - Timebox to a maximum number of days a coordinated campaign will run across all targeted users.
+  - Specify a maximum number of impressions a user should see an in-product message. Respect when a user dismisses a message.
+
+### Avoid traffic collisions
+
+- Avoid showing too many alerts at the same time. Collisions are awkward and undesirable, no more than 2 alerts a time. 
+- Consider what other alerts, teaching bubbles, or banners may be active on the page when crafting an onboarding experience.
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={8/12} border="1px solid" borderColor="red.5" mb="2" overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/89232625-b0708500-d59c-11ea-8af1-249894f6143a.png"/>
+  </BorderBox>
+  <p><Text fontWeight="bold" color="red.5">Don't:</Text> Don't forget to check coinciding feature releases on the same page.</p>
+</Flex>
+
+### Create efficient sequences
+
+Sequence tasks in a logical way to increase onboarding success. Clearly define when a user starts and successfully completes an onboarding flow. 
+
+Treat feature onboarding as a story or a journey with a beginning, middle, and end.
+
+#### Beginning: Draw attention to the feature
+
+- Invite the user to engage with an onboarding journey.
+- Use teaching bubbles, alert banners, the feature preview tray or annoucement modals to draw attention to the feature.
+
+<Flex my={3}>
+  <Box width="100%" mr="2">
+    <BorderBox border="1px solid" borderColor="green.5" mb="2" overflow="hidden">
+      <img src="https://user-images.githubusercontent.com/6744014/89225349-eb6bbc00-d58e-11ea-82b1-b84e8d35b1df.png" />
+    </BorderBox>
+    <Text fontWeight="bold" color="green.5">
+      Do:
+    </Text> Use empty states or alert banners to call attention to a feature if users may discover the feature when starting a task.
+  </Box>
+  <Box width="100%">
+    <BorderBox border="1px solid" borderColor="green.5" mb="2" overflow="hidden">
+      <img src="https://user-images.githubusercontent.com/6744014/89225996-0c80dc80-d590-11ea-9e08-b86b2022b7eb.png" />
+    </BorderBox>
+    <Text fontWeight="bold" color="green.5">
+      Do:
+    </Text> Use teaching bubbles or other easily dismissable elements if users will discover the feature when they are completing another task.
+  </Box>
+</Flex>
```

Comment:
Let's make the dos above full width and stacked, otherwise it's hard to see the image. And add alt texts.